skimage/measure/simple_metrics.py

```diff
@@ -71,12 +78,63 @@ def compare_psnr(im_true, im_test, data_range=None):
     -----
     Deprecated:
         .. versionadded:: 0.16
-
-        This function is deprecated and will be removed in scikit-image 0.18.
-        Please use the function named ``peak_signal_noise_ratio`` from the
-        ``metrics`` module instead.
+          This function is deprecated and will be removed
+          in scikit-image 0.18.
+          Please use the function named ``peak_signal_noise_ratio`` from the
+          ``metrics`` module instead.
 
     See also
     --------
     skimage.metrics.peak_signal_noise_ratio
     """
+
+
+def enhancement_measure(image: np.ndarray,
+                        size: int = 3,
+                        eps: float = 1e-6) -> float:
+    """ The image enhancement measure called EME based on [1]_.
+    It is a way of quantifying improvement of the image after enhancement.
+
+    The function uses a sliding window of user-provided size to measure
+    the mean of log of maximal and minimal intensity ratio
+    within the window.
+
+    Parameters
+    ----------
+    image : array
+        Input image of which the quality should be assessed.
+        Can be either 3-channel RGB or 1-channel grayscale.
+    size : int, optional
+        Size of the window.
+    eps : float, optional
+        Parameter to avoid division by zero.
+
+    Returns
+    -------
+    eme : float
+        The number describing image quality.
+
+    References
+    ----------
+    .. [1] Sos S. Agaian,  Karen Panetta, and Artyom M. Grigoryan.
+            "A new measure of image enhancement.",
+            IASTED International Conference on Signal Processing
+            & Communication, Citeseer, 2000,
+            :DOI:10.1.1.35.4021
```

Comment:
```suggestion
            :DOI:`10.1.1.35.4021`
```

Reply:
The DOI is incorrect. Also, backquotes must be set as suggested.

Reply:
Hi @sciunto, thanks for pointing that our!
I was having troubles with the DOI though.

The only source that lists the DOI is [this](https://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.35.4021) link, and I doubt that there is a better version or at least i couldn't find one. The [doi.org](doi.org) says this is only a prefix, but google search for doi=10.1.1.35.4021 returns the link above no problem. The original journal is not available (see [here](https://www.iasted.org/CONFERENCES/pastinfo-320.html)) so I am not sure what is the best way to prcoceed here.

Reply:
I think doi.org must be the reference. If no working doi can be found, it is better to do not display a doi. Eventually, a link to the pdf file can be added instead.

Reply:
I removed it for now.